test/github-issues/4350/issue-4350.ts

```diff
@@ -0,0 +1,55 @@
+import "reflect-metadata";
+import {closeTestingConnections, createTestingConnections} from "../../utils/test-utils";
+import {Post1, TestEnum1} from "./entity/Post1";
+import {Post2, TestEnum2} from "./entity/Post2";
+import {Connection} from "../../../src";
+import {expect} from "chai";
+
+describe("github issues > #4350 Array of enums column doesn't work at all", () => {
+    let connections: Connection[];
+    after(() => closeTestingConnections(connections));
+
+    it("should migrate postgres enums correctly", async () => {
```

Comment:
Can you add a test of going from an array to a non-array and back again?

Not sure if that's supported or not but if it worked before I'm curious if this keeps it operating.

If it didn't work before we don't need it.

Reply:
I think this is a bit out of scope for this issue. The issue itself is only about changing an existing array column. I guess there should be a test for this behaviour, but not with this pull request.

Reply:
Ok - this test needs to exist somewhere before we can merge this in - it is technically out of scope.

Unfortunately, without a test we cannot tell that this change is causing that to break or not.